src/lib/routes/admin-api/project/variants.ts

```diff
@@ -143,6 +166,17 @@ export default class VariantsController extends Controller {
             handler: this.overwriteVariantsOnEnv,
             middleware: [
                 openApiService.validPath({
+                    summary:
+                        'Create (overwrite) variants for a feature toggle under the provided environment',
+                    description: `This overwrites the current variants for the feature toggle specified in the :featureName parameter for the :environment parameter.
+                        
+                        The backend will validate the input for the following invariants:
+                        
+                        * If there are variants, there needs to be at least one variant with ${'`'}weightType: variable${'`'}
+                        * The sum of the weights of variants with ${'`'}weightType: fix${'`'} must be below 1000 (< 1000)
+                        
+                        The backend will also distribute remaining weight up to 1000 after adding the variants with ${'`'}weightType: fix${'`'} together amongst the variants of ${'`'}weightType: variable${'`'}.
```

Comment:
Same thing for the backticks here.

```suggestion
                    * If there are variants, there needs to be at least one variant with \`weightType: variable\`
                    * The sum of the weights of variants with \`weightType: fix\` must be strictly less than 1000 (< 1000)

                    The backend will also distribute remaining weight up to 1000 after adding the variants with \`weightType: fix\` together amongst the variants of \`weightType: variable\``,
```